### PR TITLE
Move raft-related tests out of store_split_test.go.

### PR DESCRIPTION
**Pull request by bdarnell**
Rename all tests in package storage_test.go to client_*_test.go
to make the division clearer.

@spencerkimball @cockroachdb/developers

### CODE REVIEW

**Comment by MycroftH** [comment]
LGTM

File: storage/client_raft_test.go [hunk 1]
```diff
@@ -0,0 +1,101 @@
+// Copyright 2014 The Cockroach Authors.
```
Comment (petermattis) [inline storage/client_raft_test.go]: Not that it matters, but it is 2015.

**Comment by petermattis** [comment]
LGTM

File: storage/client_raft_test.go (hunk 1 above)
Comment (bdarnell) [inline storage/client_raft_test.go]: Changed.

**Pull request merged**